src/structure/Header.tsx

```diff
@@ -216,7 +216,21 @@ function Header({ isDarkMode, darkModeToggle }) {
                 <RewardsHeaderLink />
               </Box> */}
               <ChainIndicator isDarkMode={isDarkMode} />
-              <Profile isDarkMode={isDarkMode} />
+              <Box
```

Comment:
<img width="882" alt="Screenshot 2024-07-16 at 12 25 13 PM" src="https://github.com/user-attachments/assets/f2834eab-bddd-4ab6-a416-9aa1ab9b9418">
Laptop view needs to be fixed